README.md

```diff
@@ -75,7 +76,13 @@ Shows details on when product will be in stock next.
 
 ### Heroku Deployment
 
-This repo includes a Procfile that can be used to deploy to heroku.
+This repo includes a `heroku.yml` that can be used to deploy to heroku.
+
```

Comment:
Instead of Runtime
s/Ubuntu/Runtime/

Reply:
Can you explain your comment I don't quite understand

Reply:
Oh, It was in line 81. Sorry I reviewed from phone.

Reply:
Added Prakash's line.